docs/packages-api.md

```diff
@@ -18,14 +18,17 @@ messages to be logged.
 
 ## run-mode
 
-A package that exports a function called `getRunMode()`, which returns one of two constants,
+A module that exports a function called `getRunMode()`, which returns one of two constants,
 depending on the value of the `CARBON_RUN_MODE` environment variable.
 
 The possible values are:
 
 - `DEV`
-- `PRODUCTION`
+- `PROD`
```

Comment:
Since we're now using this in the web-app, can you update from PRODUCTION to PROD in:

- [ ] `constants.js:7`
- [ ] `constants.js:9`

Reply:
Done!